daylite_models.gemspec

```diff
@@ -1,37 +1,39 @@
-Gem::Specification.new do |s|
-  s.name = %q{daylite-models}
-  s.version = "0.2.0"
-
-  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
-  s.authors = ["Dmitriy Dzema"]
-  s.date = %q{2014-07-25}
-  s.email = %q{dimad.ag@gmail.com}
-  s.extra_rdoc_files = ["History.txt", "Manifest.txt"]
-  s.files = ["History.txt", "LICENSE", "Manifest.txt", "README.markdown", "Rakefile", "app/models/daylite.rb", "app/models/daylite/contact.rb", "app/models/daylite/organization.rb", "app/models/daylite/organization_task_join.rb", "app/models/daylite/project.rb", "app/models/daylite/project_task_join.rb", "app/models/daylite/role.rb", "app/models/daylite/role_type.rb", "app/models/daylite/task.rb", "app/models/daylite/user.rb", "config/database.yml.exmp", "daylite_models.gemspec", "init.rb", "install.rb", "lib/active_record/connection_adapters/openbase_adapter.rb", "lib/daylite_models.rb", "tasks/users_plugin_tasks.rake", "test/daylite_models_plugin_test.rb", "test/test_helper.rb", "uninstall.rb"]
-  s.has_rdoc = true
-  s.rdoc_options = ["--main", "README.markdown"]
-  s.require_paths = ["lib"]
-  s.rubyforge_project = %q{daylite-models}
-  s.rubygems_version = %q{1.4.1}
-  s.summary = %q{Set of ActiveRecord classes to work with the Daylite 4 and 5 databases}
-  s.test_files = ["test/test_helper.rb"]
-
-  if s.respond_to? :specification_version then
-    current_version = Gem::Specification::CURRENT_SPECIFICATION_VERSION
-    s.specification_version = 2
-
-    if Gem::Version.new(Gem::RubyGemsVersion) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<activesupport>, [">= 2.0.0"])
-      s.add_runtime_dependency(%q<activerecord>, [">= 2.0.0"])
-      s.add_runtime_dependency(%q<activerecord>, [">= 0.8.3"])
-    else
-      s.add_dependency(%q<activesupport>, [">= 2.0.0"])
-      s.add_dependency(%q<activerecord>, [">= 2.0.0"])
-      s.add_dependency(%q<activerecord>, [">= 0.8.3"])
-    end
-  else
-    s.add_dependency(%q<activesupport>, [">= 2.0.0"])
-    s.add_dependency(%q<activerecord>, [">= 2.0.0"])
-    s.add_dependency(%q<activerecord>, [">= 0.8.3"])
-  end
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |spec|
+  spec.name          = 'daylite_models'
+  spec.version       = '0.5.6'
```

Comment:
Versioning between your fork and mine is not the same. I suggest we integrate all your changes and release version 1.0.0 of the main gem

Reply:
Also, let's use `DayliteModels::VERSION` here to have one single source of truth for the version in the codebase?

Reply:
Changed both in #3

Reply:
Perfect, agreed.  I would suggest making the next version 0.9.x, as there are further changes (new support, old issues fixed) that I have yet to submit.

Reply:
Once we're in agreement, then can be merged into a proper v1.0 release.

Reply:
Sure, it could be 0.3 as well. And when you are satisfied with the fixes you have done 1.0.0 could be released. I am going to port your most recent changes into #3 and merge it, I will close this PR>